thorlcr/thorutil/thormisc.cpp

```diff
@@ -392,6 +392,12 @@ CThorException *_ThorWrapException(IException *e, const char *format, va_list ar
     e->errorMessage(eStr).append(" : ");
     eStr.limited_valist_appendf(2048, format, args);
     CThorException *te = new CThorException(e->errorAudience(), e->errorCode(), eStr.str());
+    if (QUERYINTERFACE(e, IMP_Exception))
```

Comment:
minor comment.  Dynamic casts are not very efficient, so calculating me and then testing would be better.  This isn't time critical, so it doesn't matter.